oat-sa/tao-core
# Fix/tr 3390 remove calculator related arabic translations

Related to https://oat-sa.atlassian.net/browse/TR-3390

- Remove calculator related arabic translations
- Integrate new tao-core-ui, where `ln` and `log` translation were removed (https://github.com/oat-sa/tao-core-ui-fe/pull/417)

Test:

- Have a test that contains an item, where calculator test taker tool was enabled
- Launch an LTI to this delivery with `ar-arb` locale
- Check the calculator controls are not translated
- It should be LTR as well, because of https://github.com/oat-sa/tao-test-runner-qti-fe/pull/465

To help you, you can use this composer file: https://github.com/oat-sa/kitchen-playground/blob/test/tr-3390/files/delivery/composer.json
